os/board/common/README.md

```diff
@@ -0,0 +1,78 @@
+# How to partition a flash
+
+This document provides a way to partition in flash. Use CONFIG when flash partition.  
+
+## How to configure CONFIG
+
+Flash partition is provided by configurations, FLASH_PART_SIZE, FLASH_PART_TYPE and FLASH_PART_NAME. 
+FLASH_PART_SIZE represents size of each partition, FLASH_PART_SIZE does name, FLASH_PART_TYPE does interface between flash and upper layer.  
```

Comment:
Please enumerate or Itemize the three configurations above as shown below:

"Flash partition can be configured the following three parameters:
1. FLASH_PART_SIZE -- represents a partition size
2. FLASH_PART_NAME (please check above) -- represents partition name
3. FLASH_PART_TYPE -- represents interface between flash and upper layer
"

Reply:
Thanks. I modified it.